Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -190,12 +194,16 @@ public struct PackageBuilder {
         return true
     }
 
+    private var rerootedManifestPath: AbsolutePath {
+        return AbsolutePath.root.appending(component: Manifest.filename)
```

Comment:
Actually, I think what should happen is `packagePath` becomes `AbsolutePath.root` in the initializer, and we continue to use that.